### ISSUE TITLE
Edit Redirects to 404 path

### ISSUE THREAD

**Issue by rinoldsimon** [ember-learn/cli-guides#228]
Editing guides at all the pages are redirecting to 404

When I click the edit icon at the top right corner https://cli.emberjs.com/release/appendix/dev-tools/, It takes me to 
https://github.com/ember-learn/cli-guides/edit/master/guides/appendix/dev-tools.md **(404)**

Actually, it should take me to 
https://github.com/ember-learn/cli-guides/edit/main/guides/appendix/dev-tools.md

**Comment by ijlee2** [comment]
@rinoldsimon Thank you for letting us know of this issue. We recently updated the default branch name for `cli-guides` to `main`, and didn't realize the scenario with the edit link.

The edit link is hardcoded in `guidemaker-ember-template`. Possible solutions are:

- Release a fix in [`guidemaker-ember-template`](https://github.com/ember-learn/guidemaker-ember-template/blob/master/app/templates/components/guides-article.hbs#L11)
- Wait for GitHub to [support redirecting edit links](https://github.blog/changelog/2020-07-17-links-to-deleted-branches-now-redirect-to-the-default-branch/)
- Something else

**Comment by jenweber** [comment]
I think we can add this to Guidemaker as an addon config pretty easily. I’ll try to look at it today or ask Chris for help when we pair on Thursday.

A third option is to recreate the “master” branch so that the links go somewhere. I think I will do that as well, while we work on the other task. How does that sound, @ijlee2 ?

**Comment by ijlee2** [comment]
@jenweber Yep, a temporary revert of the branch name is okay with me. Thanks for your help!

**Comment by jenweber** [comment]
We don’t have to revert anything, thankfully. I will branch from main, call the new branch master, and leave main as the default. I’ll do that now. I’ll also make sure that removing it is tracked in an issue.

**Comment by jenweber** [comment]
Step one is done and the links work again.

**Comment by Windvis** [comment]
This commit seems to add the needed configuration: https://github.com/ember-learn/cli-guides/commit/2b376bcd9e3d9d1eae4eb3a669221faed9a16017

So I think this issue can be closed?
